cpp/deplex/src/deplex/plane_extractor.cpp

```diff
@@ -73,6 +74,7 @@ class PlaneExtractor::Impl {
   int32_t image_height_;
   int32_t image_width_;
   Eigen::MatrixXi labels_map_;
+  std::vector<std::vector<Eigen::Index>> neighbours;
```

Comment:
1. Why do we need precomputed `neighbours` 
2. Why is it a field of `PlaneExtractor`? It doesn't make sense for me.

Reply:
To get rid of numerous calls to the getNeighbours function, in which we look at neighboring cells every time and initialize a vector with them, when processing each image